README.md

```diff
@@ -157,12 +158,15 @@
 * [mysql](https://www.reddit.com/r/mysql)
 * [PostgreSQL](https://www.reddit.com/r/PostgreSQL/) - For PostgreSQL database enthusiasts.
 * [redis](https://www.reddit.com/r/redis/) - Redis community subreddit.
-* [RethinkDB](https://www.reddit.com/r/rethinkdb/) 
+* [RethinkDB](https://www.reddit.com/r/rethinkdb/)
 * [SQLServer](https://www.reddit.com/r/SQLServer/) - Microsoft SQL Server Administration and T-SQL Programming.
 
+## Editors
 
+* [emacs](https://www.reddit.com/r/emacs)
+* [vim](https://www.reddit.com/r/vim)
 
-### Entertainment
+## Entertainment
```

Comment:
Why did you remove a hash?

Reply:
because in the table of contents it appears as a "top level" entry. I thought it was a typo. 

Should I put it back to 3?

Reply:
Oh yeah, you are right! It's ok. Merging 👍